next/src/pages/Sponsors2.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useState } from "react";
+import Image from "next/image";
+import Footer from "../components/footer/Footer";
+import Navbar from "../components/navbar/Navbar";
+import { NavbarOpenHandler, NavbarType } from "../components/navbar/types";
+import styled from 'styled-components'
+import Link from 'next/link'
+import { device } from '../styles/device'
+
+
+import Otter from '../svgs/otter.png'
+
+const Text = styled.p`
```

Comment:
all these styled components can be moved into a different file and then imported into this one!